test/server/cards/07-GR/RoragScreamer.spec.js

```diff
@@ -0,0 +1,51 @@
+describe('Rorag Screamer', function () {
+    describe("Rorag Screamer's ability", function () {
+        beforeEach(function () {
+            this.setupTest({
+                player1: {
+                    house: 'brobnar',
+                    hand: ['rorag-screamer'],
+                    inPlay: ['cpo-zytar', 'alaka']
+                },
+                player2: {
+                    inPlay: ['dew-faerie', 'cornicen-octavia']
+                }
+            });
+        });
+
+        it('readies after fighting', function () {
+            this.player1.playCreature(this.roragScreamer);
+            this.player1.endTurn();
```

Comment:
you can do `this.roragScreamer.exhausted = false` to avoid going through so much code.

Reply:
I knew about this but I'm going through the turn stuff on purpose because it feels more realistic to me and I don't want to accidentally miss anything important that happens when readying.  I prefer this way but if you feel strongly about it I can change it.